easybuild/easyconfigs/a/argtable/argtable-2.13-foss-2015b.eb

```diff
@@ -0,0 +1,25 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'ConfigureMake'
+
+name = 'argtable'
+version = '2.13'
+
+homepage = 'http://argtable.sourceforge.net/'
+description = """ Argtable is an ANSI C library for parsing GNU style 
+ command line options with a minimum of fuss. """
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%s%s.tar.gz' % (name, version.replace('.','-'))]
```

Comment:
nitpic all you like - I enjoy the fact that someone is taking the time to look so closely at the code!